src/filters/adjust/adjustgamma.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @filter           Gamma Adjustment
+ * @description      Gamma correction using a given gamma
+ * @param gamma      Gamma exponent between 0 and 5
+ */
+function adjustGamma(gamma) {
+    gl.adjustGamma = gl.adjustGamma || new Shader(null, '\
+        uniform sampler2D texture;\
+        uniform float gamma;\
+        varying vec2 texCoord;\
+        void main() {\
+            vec4 color = texture2D(texture, texCoord);\
+            color.rgb = vec3(pow(color.r, gamma), pow(color.g, gamma), pow(color.b, gamma));\
```

Comment:
Can you please try this:
```suggestion
            color.rgb = pow(color.rgb, vec3(gamma));\
```

Reply:
Changed it, works as well.